src/Domain/Asset/ValueObject/UploadCompletionProof.php

```
@@ -0,0 +1,23 @@
+use App\Domain\Asset\UploadCompletionProofSource;
+
+final readonly class UploadCompletionProof
+{
+    public string $name;
+
+    public function __construct(string $name, public UploadCompletionProofSource $source)
+    {
+        $normalizedName = trim($name);
+
+        if ($normalizedName === '') {
+            throw new \InvalidArgumentException('Upload completion proof name cannot be empty');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '18', column '23'). (undefined)

(MissingImport)